src/libraries/System.Runtime/tests/System/Reflection/ModuleTests.cs

```diff
@@ -81,6 +81,9 @@ public void CustomAttributes()
         [Fact]
         public void FullyQualifiedName()
         {
+#if SINGLE_FILE_TEST_RUNNER
```

Comment:
It does not sound right for the test to contain ifdefs like this. It should be a dynamic check instead.

Reply:
This is already merged with https://github.com/dotnet/runtime/pull/50996 - I cherry picked it because I'm tired of working in two repos.

We don't have a way to build a dynamic check that wouldn't just result in testing a tautology ("on a runtime that returns `<Unknown>` for FullName, check that FullName returns `<Unknown>`").